src/templates/installing/os/ubuntu/mongo/install.md

```diff
@@ -1,8 +1,10 @@
 The following is an abbreviation of the official [MongoDB installation guide for Ubuntu](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/). If you're having issues, fall back to using that guide instead.
 
 ```bash
-wget -qO - https://www.mongodb.org/static/pgp/server-{{versions.recommended.mongo}}.asc | sudo apt-key add -
-echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/{{versions.recommended.mongo}} multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-{{versions.recommended.mongo}}.list
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
```

Comment:
The `versions.recommended.mongo` variables was removed in this change set. Please restore it and update the variable as necessary.

Reply:
I see that a suitable request has already been created
https://github.com/NodeBB/docs/pull/86
So mine is redundant